doc/source/raysgd/v2/user_guide.rst

```diff
@@ -279,29 +620,26 @@ A couple caveats:
 
     analysis = tune.run(trainable, config={
         "lr": tune.uniform(), "batch_size": tune.randint(1, 2, 3)}, num_samples=12)
+..
+    Advanced APIs
+    -------------
 
+    TODO
 
-Distributed metrics (for Pytorch)
----------------------------------
+    Training Run Iterator API
+    ~~~~~~~~~~~~~~~~~~~~~~~~~
 
-In real applications, you may want to calcluate optimization metrics besides accuracy and loss: recall, precision, Fbeta, etc.
+    TODO
 
-RaySGD natively supports `TorchMetrics <https://torchmetrics.readthedocs.io/en/latest/>`_, which provides a collection of machine learning metrics for distributed, scalable Pytorch models.
+    Stateful Class API
+    ~~~~~~~~~~~~~~~~~~
 
-Here is an example:
-
-.. code-block:: python
-
-    import torch
-    import torchmetrics
-    import ray
+    TODO
 
-    def train_func(config):
-        preds = torch.randn(10, 5).softmax(dim=-1)
-        target = torch.randint(5, (10,))
+.. _sgd-backwards-compatibility:
 
-        acc = torchmetrics.functional.accuracy(preds, target)
-        ray.sgd.report(accuracy=acc)
+..
+    Backwards Compatibility
+    -------------
```

Comment:
Can you align this so it's rendered properly?

Reply:
I left this indented as a comment for now. Planning to add this in after Class API is implemented and we can include an example.